### PR TITLE
Added the reviews

### LINKED ISSUE #127: Implement user rating
**Is your feature request related to a problem? Please describe.**
Currently the user cannot rate the services implement it such that userr can rate

Take the idea of implementation from the susi_skills_cms repo

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
Are you working on this?
Working on this

### PR DESCRIPTION
**Pull request by ritwizsinha**
Fixes #127 

Changes:
Added a review for modal (without a backend for now)

Surge Deployment Link: https://pr-158-sahaayak.surge.sh

Screenshots for the change: [Add here screenshots showing the changes.]
image

image

### CODE REVIEW

File: src/components/Review/Review.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                <Button key="back" onClick={this.handleCancel}>
+                  Return
+                </Button>,
+                <Button key="submit" type="primary" loading={loading} onClick={this.handleOk} disabled={rating===null?true:false}>
+                  Submit
+                </Button>,
+              ]}
+            >
+              {rating===null ?
+            (<Alert type="info" message="Please atleast rate the service before submitting" />):null}
```
Comment (harsh-jindal) [inline src/components/Review/Review.js]: Not required. Just keep the submit button disabled.

Comment (harsh-jindal) [inline src/components/Review/Review.js]: Not required. Just keep the submit button disabled.

File: src/components/Review/Review.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      handleCancel = () => {
+        this.setState({ visible: false
+         });
+      };
+    
+    render() {
+        const { visible, loading,rating } = this.state;
+        // console.log(rating);
+        const {error,review}  = this.props;
+        // console.log(review);
```
Comment (harsh-jindal) [inline src/components/Review/Review.js]: Remove comments

File: src/redux/actions/reviewActions.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    try {
+      dispatch({
+        type: POST_REVIEW_LOADING
+      });
+      const res = await Axios.post(
+        `${url}/reviews/${id}`,
+        reviewData,
+        tokenConfig(getState)
+      );
+      console.log(res);
```
Comment (harsh-jindal) [inline src/redux/actions/reviewActions.js]: Remove console.log

File: src/components/Review/Review.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        console.log(error,review);
+        return (
+          <div id="review">
+            
+            <Button type="primary" onClick={this.showModal}  style={{margin:20}}>
+              Review the Service
```
Comment (harsh-jindal) [inline src/components/Review/Review.js:54]: Change this to `Add review`

File: src/components/Review/Review.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+            </Button>
+            <Modal
+              visible={visible}
+              title="Review the service"
+              onOk={this.handleOk}
+              onCancel={this.handleCancel}
+              footer={[
+                <Button key="back" onClick={this.handleCancel}>
+                  Return
```
Comment (harsh-jindal) [inline src/components/Review/Review.js:63]: change this to `Cancel`

File: src/components/Review/Review.js [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+                </Button>,
+                <Button key="submit" type="primary" loading={loading} onClick={this.handleOk} disabled={rating===null?true:false}>
```
Comment (harsh-jindal) [inline src/components/Review/Review.js]: rename the function `this.handleOk` to `this.handleSubmit`

**Comment by harsh-jindal** [comment]
@ritwizsinha Just make these minor changes

File: src/components/Review/Review.js (hunk 1 above)
Comment (ritwizsinha) [reply src/components/Review/Review.js]: done

File: src/components/Review/Review.js (hunk 2 above)
Comment (ritwizsinha) [reply src/components/Review/Review.js]: done

File: src/redux/actions/reviewActions.js (hunk 3 above)
Comment (ritwizsinha) [reply src/redux/actions/reviewActions.js]: done

File: src/components/Review/Review.js (hunk 4 above)
Comment (ritwizsinha) [reply src/components/Review/Review.js:54]: done

File: src/components/Review/Review.js (hunk 5 above)
Comment (ritwizsinha) [reply src/components/Review/Review.js:63]: done

File: src/components/Review/Review.js (hunk 6 above)
Comment (ritwizsinha) [reply src/components/Review/Review.js]: done

**Comment by harsh-jindal** [comment]
@ritwizsinha I'll merge this but you have left a lot of empty lines. Also always remove comments and console logs from the code. Also learn how to make only 1 commit. I'll write the steps here:
1. git checkout branch-name
2. git fetch upstream
3. git rebase upstream/master
4. git reset HEAD~
5. make your changes
6. git commit -am "commit message"
7. git push -f origin branch-name

If you get conflicts in 3rd step:
1. Go to that file resolve conflicts.
2. Add the conflicting file(s)
e.g git add src/components/Admin/Admin.js
3. git rebase --continue

Again follow from 4th step if you want to make changes.
Or directly execute the 7th step

**Pull request merged**